code/game/objects/items.dm

```diff
@@ -715,7 +715,7 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 		return
 
 	if(user.client)
-		user.client.view_size.set_view_radius_to(viewsize/2-2)//sets the viewsize to reflect radius changes properly
+		user.client.view_size.set_width_and_height(viewsize, viewsize)
 
 		var/tilesize = 32
```

Comment:
Does not support widescreen properly

Reply:
it does, this var only sets the width and height var which ARENT the view size, they just increase/reduce the viewsize with its values, infact its most times at 0.